docs/dev_guide/tests.rst

```diff
@@ -245,31 +245,21 @@ Here is a simple example: ::
     def test_simple_plot():
         plt.plot([0,1])
 
-The current figure at the end of the unit test, or an explicitly returned figure, has its hash compared against an established hash library (more on this below).
+The current figure at the end of the unit test, or an explicitly returned figure, has its hash (currently ``SHA256``) compared against an established hash collection (more on this below).
 If the hashes do not match, the figure has changed, and thus the test is considered to have failed.
 
-You will need to update the library of figure hashes after you create a new figure test or after a figure has intentionally changed due to code improvement.
-The file is located at "sunpy/tests/figure_tests_env_py36.json".
-
-Furthermore, to run the figure tests, you need to have the same Python version as the figure environment.
-Otherwise, the tests will be skipped.
-Since the hashes are checked, we test figures against a fixed set of packages and the best way to check this is to use tox::
+To run the figure tests you need to be very careful, as any pixel that has changed, will change the hash.
+In order to avoid changes due to package versions, we recommend using tox::
 
     $ tox -e figure
 
-or::
-
-    $ pytest -m "figure"
-
-These will only run the figure tests and nothing else.
-
-If you have a Python environment that matches the base Python version used in the figure environment, these tests will run and probably fail.
-To avoid running the tests::
-
-    $ pytest -m "not figure"
+This will ensure that any figures created are checked using the current packages we test against.
```

Comment:
```suggestion
This will ensure that any figures created are checked using the package versions that were used to create the original figure hashes.
```